### PR TITLE
Abstracted booking card component and revamped

### PR DESCRIPTION
**Pull request by YongbinWang**
BookingCard_Bef
BookingCard_After2 0

### CODE REVIEW

**Review by shaokiat — APPROVED** [review]
LGTM

File: frontend/src/common/get24HourTime.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @param eventStartTime date time number
+ * @returns fomatted time in form of HH:MM:SS  (eg, 19:00:00 or 12:10:20)
```
Comment (hui444) [inline frontend/src/common/get24HourTime.ts:4]: pls update comment!

**Review by hui444 — CHANGES_REQUESTED** [review]

File: frontend/src/common/get24HourTime.ts (hunk 1 above)
Comment (YongbinWang) [reply frontend/src/common/get24HourTime.ts]: Resolved in commit 4c7366045b4b24d42221ea1f482ddfc9977ba8c7

File: frontend/src/common/get24HourTime.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @param eventStartTime date time number
```
Comment (hui444) [inline frontend/src/common/get24HourTime.ts:3]: consider calling this param something more generic like dateTime or unixDate etc

**Review by hui444 — APPROVED** [review]
minor issue w the param name but otherwise lgtm!

File: frontend/src/common/get24HourTime.ts (hunk 2 above)
Comment (YongbinWang) [reply frontend/src/common/get24HourTime.ts]: Resolved in commit c250da81f5f68f938da6eb63dc522128055245a8

**Comment by YongbinWang** [comment]
[Create booking card component](https://trello.com/c/PSPv6xhJ/9-create-booking-card-component)

**Pull request merged**